src/app/server/Mdns.h

```diff
@@ -18,6 +18,7 @@
 #pragma once
 
 #include <core/CHIPError.h>
+#include <stddef.h>
```

Comment:
Is this needed? Doesn't look like anything new in the header required it..

Reply:
its needed for size_t:

CHIP_ERROR GenerateRotatingDeviceId(char rotatingDeviceIdHexBuffer[], size_t rotatingDeviceIdHexBufferSize);